pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/UploadRefreshDeleteIntegrationTest.java

```diff
@@ -153,6 +210,16 @@ public void testRefresh(String tableName, SegmentVersion version) throws Excepti
     verifyNRows(nRows2, nRows2+nRows3);
   }
 
+  @Test(dataProvider = "configProvider")
```

Comment:
No need for the data provider since both tableName and version are not used. Remove the parameter inside the testRetry()